test/unit/modules.spec.js

```diff
@@ -668,6 +668,37 @@ describe('Modules', () => {
         store.state
       )
     })
+
+    it('action before/after subscribers', (done) => {
+      const beforeSpy = jasmine.createSpy()
+      const afterSpy = jasmine.createSpy()
+      const store = new Vuex.Store({
+        actions: {
+          [TEST]: () => new Promise(resolve => resolve())
```

Comment:
This can be simplified with `Promise.resolve()` :slightly_smiling_face:

Reply:
oh duh. Will update!